### DISCUSSION TITLE
how to change the state on click

### DISCUSSION THREAD

**Discussion by Prasanna-kumar-Patra** [splinetool/react-spline#108] [Q&A]
how to change the state to a specific state on click in React

**Comment by marcofugaro** [comment]
See the [Trigger Spline events from outside](https://github.com/splinetool/react-spline#trigger-spline-events-from-outside) section.

**Reply by jamesvclements** [reply]
Does this mean you have to create an Event within Spline to change the state, and then trigger that event? Or is there an API like `setState('Active')` to switch the element to a state named Active?
